lib/chef-dk/command/generator_commands/cookbook.rb

```diff
@@ -56,6 +57,13 @@ class Cookbook < Base
           boolean:      true,
           default:      true
 
+        option :skip_prompt,
```

Comment:
Pwehaps we should mirror the `-y` option provided in knife since thats what many people are used to.